src/app/constants/paths.ts

```diff
@@ -0,0 +1,9 @@
+export const Paths = {
+    Login       : 'login',
+    Signup      : 'signup',
+    Admin       : 'admin',
+    Profile     : 'profile',
+    UserProfile : 'profile/:id',
+    ProfileEdit : 'profile/:id/edit',
+    Edit        : 'edit'
```

Comment:
i don't think that spacing is needed between the values and the objects and also this can be made enum